recipe/recipe.yaml

```
@@ -12,24 +12,29 @@ source:
   patches: 0001-use-imported-getopt.patch
 
 build:
-  number: 2
+  number: 3
+  variant:
+    use_keys:
+      - x86_64_microarch_level
+    down_prioritize_variant: ${{ 0 if x86_64_microarch_level == 1 else 1 }}
+
   script:
     # Get an updated config.sub and config.guess
-    - if: unix
-      then: cmake "${CMAKE_ARGS}" -DSISCONE_BUILD_EXAMPLES=OFF -DSISCONE_ENABLE_DEBUG=OFF -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
     - if: win
       then: cmake %CMAKE_ARGS% -DSISCONE_ENABLE_DEBUG=OFF -S . -B build
+      else: cmake ${CMAKE_ARGS} -DSISCONE_BUILD_EXAMPLES=OFF -DSISCONE_ENABLE_DEBUG=OFF -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
     - cmake -LH build
-    - if: unix
-      then: cmake --build build --parallel="${CPU_COUNT}"
     - if: win
       then: cmake --build build --config=Release --parallel="%CPU_COUNT%"
+      else: cmake --build build --parallel="${CPU_COUNT}"
     - cmake --install build
 
 requirements:
   run_exports:
     - ${{ pin_subpackage('siscone', upper_bound='x.x') }}
   build:
+    - if: unix and x86_64
+      then: x86_64-microarch-level ==${{ x86_64_microarch_level }}
```

```suggestion
      then: x86_64-microarch-level ${{ '==' ~ x86_64_microarch_level if x86_64_microarch_level else "*" }}
```

@lgray I think we need this given a `rattler-build` bug that we saw in `awkward-cpp`. c.f. https://github.com/prefix-dev/rattler-build/issues/1778

---

Siscone releases so infrequently, let's see if this gets fixed in two weeks.